conf/application.conf

```diff
@@ -205,10 +205,3 @@ urls {
 accessibility-statement.service-path = "/plastic-packaging-tax-registration"
 
 go-live-date = "2022-04-01"
-
-# Feature Flags
-# Generic Feature Flags
-features {
-  partnershipEnabled = true
```

Comment:
dont forget the differnt configs once this is deployed 
https://github.com/hmrc/app-config-production/blob/main/plastic-packaging-tax-registration-frontend.yaml#L53